app-text/xpdf/xpdf-3.04.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="Open source Portable Document Format (PDF) viewer"
+DESCRIPTION="The Xpdf project also includes a PDF text extractor, PDF-to-PostScript converter, \
```

Comment:
You can replace
```
DESCRIPTION="The Xpdf project also includes a PDF text extractor, PDF-to-PostScript \
converter, and various other utilities.
```
by
```
DESCRIPTION="The Xpdf project also includes a PDF text extractor, \
PDF-to-PostScript converter, and various other utilities.
```
but IMO the following description might be slighty better:
```
DESCRIPTION="Xpdf is a light-weight open souce viewer for Portable Document \
Format (PDF) files. The Xpdf project also includes a PDF text extractor, \
PDF-to-PostScript converter, and various other utilities."
```